src/icalendar/tests/test_fixed_issues.py

```diff
@@ -75,35 +75,14 @@ def test_issue_82(self):
             b"VALUE=BINARY:dGV4dA==\r\nEND:VEVENT\r\n"
         )
 
-    def test_issue_101(self):
-        """Issue #101 - icalendar is choking on umlauts in ORGANIZER
-
-        https://github.com/collective/icalendar/issues/101
+    def test_issue_100(self):
```

Comment:
Do you know where these new tests come from? I am a bit confused here. They do not seem to be in master. Did you add them?

- [ ] `test_issue_100`
- [ ] `test_issue_70`

Reply:
That was me! I added them...

Reply:
I merged 
https://github.com/collective/icalendar/pull/406/files#diff-308383c09e03bc2f4fe0e55272b1dcd4dc5217cb1a825d203609a833414cbd94 - 70
https://github.com/collective/icalendar/pull/408/files#diff-308383c09e03bc2f4fe0e55272b1dcd4dc5217cb1a825d203609a833414cbd94L78 - 100
and now, I added it again... sorry!